---
layout: page
title: 
subtitle: 
---
<p align="center">
<img src="/img/workshops/phylogeny_logo.png" width="500">
</p>


[Slides](https://github.com/vinni-bio/WS-20160909/tree/master/Slides)<br/>
[Tutorials](https://github.com/vinni-bio/WS-20160909)



Phylogenetic inference has become a core part of any modern
evolutionary analysis and has found important applications in
biomedical research, ecology, microbiology, to cite only a few
fields. To adapt and leverage the deluge of genomics data that is
commonly generate in high-throughput sequencing methods, new
phylogenetics methods are continuously proposed. To familiarize users
with popular approaches and new development in this field, [UH
Cyberinfrastructure](http://www.hawaii.edu/its/ci/ "Cyberinfrastructure") Bioinformatice Core, in conjunction with the
Centers of Biomedical Research Excellence ([COBRE-PCEIDR](http://pceidr.jabsom.hawaii.edu/ "COBRE-PCEID")) and the
Research Centers for Minority Institutions ([RCMI-BRIDGES](http://rcmi-bridges.jabsom.hawaii.edu/ "RCMI-BRIDGES")) is offering
a half-day introductory workshop on phylogenetics and
phylogenomics. This short overview is targeted towards graduate
students, post-docs, researchers and faculty members of the University
of Hawaii System.

The workshop will review a broad range of topics ranging from
elementary concepts such as sequence alignment, and evolutionary model
selection, to more advanced ones such as tree calibration. The
workshop contains a hands-on component during which users will be
building, optimizing and visualizing trees using a suite of popular
tools.

A detailed workshop schedule is available [here](https://www.dropbox.com/s/9nobek34rcbkcvp/PHYLOGENY_WORKSHOP_SCHEDULE.docx?dl=0 "Workshop Schedule"). Please note that this schedule is subject to change.

### Required and recommended tools

#### Required Programs

| Program |URL|
| -------------|:-------------|
| MEGA      	 	| [http://www.megasoftware.net/](http://www.megasoftware.net/)						|
| Sequence Matrix      	| [http://gaurav.github.io/taxondna/](http://gaurav.github.io/taxondna/)					|
| MrBayes      		| [http://mrbayes.sourceforge.net/download.php](http://mrbayes.sourceforge.net/download.php)				|
| jmodeltest      	| [http://goo.gl/lswTBG](https://drive.google.com/drive/folders/0ByrkKOPtF_n_OUs3d0dNcnJPYXM)	|
|  BEAST 2.4.*      	| [http://beast2.org/](http://beast2.org/) |
|  Tracer      		| [http://tree.bio.ed.ac.uk/software/tracer/](http://tree.bio.ed.ac.uk/software/tracer/) |
|  FigTree      	| [http://tree.bio.ed.ac.uk/software/figtree/](http://tree.bio.ed.ac.uk/software/figtree/) |
|  PartitionFinder      | [http://www.robertlanfear.com/partitionfinder/](http://www.robertlanfear.com/partitionfinder/) |


#### Recommended Programs

| Program     |URL|
| -----------------------------|:-------------|
| TextWrangler (<small>Mac OS X</small>)  | [http://www.barebones.com/products/textwrangler/](http://www.barebones.com/products/textwrangler/) |
| Notepad++ (<small>Windows</small>)      | [https://notepad-plus-plus.org](https://notepad-plus-plus.org) |
| RAxML                    | [http://sco.h-its.org/exelixis/web/software/raxml/index.html](http://sco.h-its.org/exelixis/web/software/raxml/index.html) |



### Date and Location

The workshop September 9th, 2016 from 9:00 AM to 1:00 PM in the Information Technology Center building (ITS) on Manoa Campus (2520 Correa Road, Honolulu, Hawaii 96822), room 105A.

### For more information, please contact: 

Kirill Vinnikov: vinni \<at\> hawaii edu <br/>
or <br/>
Mahdi Belcaid: mahdi \<at\> hawaii  edu

### Registration

This workshop is limited to 16 participants. To reserve a seat, please signup below.

<div class="cognito">
<script src="https://services.cognitoforms.com/s/lsYMFXl4X06ptGHB72ODFA"></script>
<script>Cognito.load("forms", { id: "1" });</script>
</div>


